packages/playwright-core/src/server/page.ts

```diff
@@ -189,6 +192,12 @@ export class Page extends SdkObject {
     if (delegate.pdf)
       this.pdf = delegate.pdf.bind(delegate);
     this.coverage = delegate.coverage ? delegate.coverage() : null;
+    this._instrumentationListener = {
+      onBeforeCall(sdkObject, metadata) {
```

Comment:
Is there an `onAfterCall` as well so we can have start and end annotations?

Reply:
Yes, I've taken it out for now because it created problems (playwright crashed or hanged, perhaps because my `onAfterCall` callback tried to send a CDP command after the page was closed) but will add it back.